main.go

```diff
@@ -37,8 +45,17 @@ func main() {
 	flag.StringVar(&cpuProfile, "cpuprofile", "", "write cpu profile to `file`")
 	flag.StringVar(&memProfile, "memprofile", "", "write memory profile to `file`")
 	flag.StringVar(&metricsAddr, "metrics-addr", "", "ip:port bind for prometheus metrics")
+	flag.BoolVar(&version, "version", false, "show version and exit")
 	flag.Parse()
 
+	if version {
+		fmt.Printf("Coraza SPOA Version: %s\n", Version)
+		if bi, ok := debug.ReadBuildInfo(); ok {
```

Comment:
Maybe using https://pkg.go.dev/runtime/debug#BuildSetting instead?

Reply:
Huh? This is exactly what that is doing 😄